generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -157,17 +157,11 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
         this.activeModal.dismiss(result);
     }
 
-    private onSaveError(error) {
-        try {
-            error.json();
-        } catch (exception) {
-            error.message = error.text();
```

Comment:
I think this is to handle some cases where the message is text instead of json so that when the error is intercepted in global handle the `error.message` attribute has some value if we are removing this then we need to make sure similar handling is done in global error handler